backend/app/services/project_automation_execution.py

```
@@ -399,11 +401,16 @@ def _managed_prompt(
             ),
             "请读取候选执行者并按调度要求完成分派，不要执行任务。",
         ]
-        instruction = (rule.description or "").strip()
+        instruction = self._run_instruction(rule, run).strip()
```

Ruff (0.16.1)

[error] Undefined name `self`

(F821)